templates/briqpay-checkout.php

```diff
@@ -38,7 +47,9 @@
 		</div>
 		<div id="briqpay-iframe-wrapper">
 			<?php do_action( 'briqpay_wc_before_snippet' ); ?>
-			<?php briqpay_wc_show_snippet(); ?>
+			<?php
+			( true === $show_snippet ) ? briqpay_wc_show_snippet() : wc_print_notice( 'The Briqpay Credentials are incorrect! Please enter the valid credentials and try again.', 'error' );
```

Comment:
Maybe change this to wc_add_notice instead? Will make it show in the correct notice place as well that WooCommerce adds